README.md

```diff
@@ -1,44 +1,41 @@
-# Your First HTML Tag
+# Your First HTML Tag 
```

Comment:
Formalism:

* For the moment the standard says Problem Statement. Can we get everything the same and then retroactively change later?
* I'm pretty keen on saying "Objective" over "Students..." I'm open to change but want it to happen at the standard level and THEN percolate thru, not a series of 1/offs.
* I want the objectives to be the titles of the h2s. I want it DEAD OBVIOUS where to go if you don't meet an objective.

Reply:
Also: Please trim trailing w/s, thanks.